### ISSUE TITLE
Issue with downloading package

### ISSUE THREAD

**Issue by bquinby** [ecospat/ecospat#24]
When I attempt to install ecospat I get the following response...
Warning in install.packages :
  installation of package ‘ecospat’ had non-zero exit status

The downloaded source packages are in
	‘C:\Users\brand\AppData\Local\Temp\RtmpyoIGHw\downloaded_packages’
> library(ecospat)
Error in library(ecospat) : there is no package called ‘ecospat’

I have attempted to reinstall R multiple times and I run into the same issue. Any help would be greatly appreciated.

**Comment by bpetitpi** [comment]
Hi, 

What's your OS and R version ?

**Comment by bquinby** [comment]
Hi,

I figured out that the issue was a campus network error and nothing on my end as far as OS or R version. I have successfully installed Ecospat and had it running all weekend. Thank you for reaching out!


Brandon M. Quinby, Ph.D.

Post Doc Research Assistant

Department of Forestry and Natural Resources

Purdue University

715 West State Street

West Lafayette, IN 47907

765-413-8141 (phone)

Pronouns: he, him, his
